src/main/java/module-info.java

```diff
@@ -9,12 +9,12 @@
     // JavaFX
     requires javafx.base;
     requires javafx.graphics;
-    requires javafx.swing;
```

Comment:
oh wow, this is great! - No more swing dependencies!

Reply:
Huh, very interesting, since the undomanager is still the swing one.